Protos/V1/events_common.proto

```diff
@@ -3,15 +3,31 @@ syntax = "proto3";
 package armonik.api.grpc.v1.events;
 
 import "result_status.proto";
+import "results_filters.proto";
 import "task_status.proto";
+import "tasks_filters.proto";
 
 option csharp_namespace = "ArmoniK.Api.gRPC.V1.Events";
 
+/**
+ * Represents the events that can be returned in the EventSubscriptionResponse
+ */
+enum EventsEnum {
```

Comment:
From what we've experienced in Angular, I think we should avoid having naked enums used directly. I would recommend to wrap it into its own message before using it.

Reply:
It depends on where it is used. This is not a case that cause problems. @esoubiran-aneo your thought ?

Reply:
IIRC, there are issues when you put enums directly as part of oneofs. In this case, you have to wrap them into messages

Reply:
Yes, works fine in angular. (issue comes when wrapped in a message)